src/components/shared/forms/ProjectDetailsForm.tsx

```diff
@@ -5,7 +5,7 @@ import { FormItems } from 'components/shared/formItems'
 import { normalizeHandle } from 'utils/formatHandle'
 import { cidFromUrl, unpinIpfsFileByCid } from 'utils/ipfs'
 
-export type ProjectFormFields = {
+export type ProjectDetailsFormFields = {
```

Comment:
nice idea trying to share this form, i think that'll work nicely

Reply:
Cheers, confirmed in https://github.com/jbx-protocol/juice-interface/pull/533 that, with a bit more work, it's totally re-usable 🎉